### PR TITLE
Showcase window duration

### PR DESCRIPTION
**Pull request by yigitozgumus**
- TooltipViewStateFactory parameters are updated.
- showDuration variable and its builder function are added to define visibilityDuration.
- isShowcaseViewVisibleIndefinitely variable and its builder function are is added to dismiss the view after showDuration is passed.
- Sample button for the feature is added.

### CODE REVIEW

File: library/src/main/java/com/trendyol/showcase/ui/showcase/ShowcaseActivity.kt [hunk 1]
```diff
@@ -22,6 +24,11 @@ class ShowcaseActivity : AppCompatActivity() {
                 }
             }
             setContentView(view)
+            if (model.isShowcaseViewVisibleIndefinitely.not()) {
+               Handler(Looper.getMainLooper()).postDelayed({
+                   finishShowcase(ActionType.EXIT)
```
Comment (bilgehankalkan) [inline library/src/main/java/com/trendyol/showcase/ui/showcase/ShowcaseActivity.kt:29]: What if user closes the showcase before this postDelayed block called?

File: README.md [hunk 2]
```diff
@@ -93,7 +93,8 @@ showcaseManager.show(context) // or showcaseManager.show(context, REQUEST_CODE_S
 | `builder.setSlidableContentList(List<SlidableContent>)`            | show slidable content                | yes      | null                         | no       |  
 | `builder.build()`                              | will return ShowcaseManager instance                     | no       |                              |          |
 | `showcaseManager.show(Context)`                | show the tooltip with set attributes on                  | no       |                              |          |
-
+| `builder.showDurationMillis(Long)`             | duration of the tooltip visibility                       | yes       | 2000L                       | no       |
```
Comment (bilgehankalkan) [inline README.md:96]: You can move this lines above `builder.build()` line.

File: library/src/main/java/com/trendyol/showcase/ui/showcase/ShowcaseActivity.kt (hunk 1 above)
Comment (yigitozgumus) [reply library/src/main/java/com/trendyol/showcase/ui/showcase/ShowcaseActivity.kt]: There is no visible effect but the callback should be removed. Done [here](https://github.com/Trendyol/showcase/pull/26/commits/2fb95bc9041ea7aa5c774696cc436bc959a4424e)

File: README.md (hunk 2 above)
Comment (yigitozgumus) [reply README.md:94]: Done [here](https://github.com/Trendyol/showcase/pull/26/commits/08054099a6c2d11f0ba3ae4f984fd120b0b78b86)

**Review by ozanturcan — APPROVED** [review]

**Review by mucahidkambur — APPROVED** [review]

**Review by kadiraltinok — APPROVED** [review]

**Pull request merged**